src/scene/forward-renderer.js

```diff
@@ -2165,9 +2165,7 @@ pc.extend(pc, function () {
                         parameters = material.parameters;
                         for (paramName in parameters) {
                             parameter = parameters[paramName];
-                            if (!parameter.scopeId) {
-                                parameter.scopeId = device.scope.resolve(paramName);
-                            }
+                            parameter.scopeId = device.scope.resolve(paramName);

```

Comment:
@guycalledfrank 

Thanks for your review.

This issue is caused by old value remaining in global variable(pc.ModelHandler.DEFAULT_MATERIAL).
graphicsDevice ScopeSpace always has latest ScopeId.
So always get ScopeId reference from ScopeSpace is more safer.